### PR TITLE
Update README, Developer Productivity Stack.

### PR DESCRIPTION
**Pull request by OliverMorenoGonzalez**
### For all PRs

[ ] I read the [Contribution Guide](CONTRIBUTING.md)

### For new stacks

[ ] I followed the [do's and don'ts](CONTRIBUTING.md#contributing-dos-and-donts)
[ ] This stack does not already exist

### CODE REVIEW

**Comment by jdorfman** [comment]
Hey @OliverMorenoGonzalez 

Thanks for submitting! I will get this reviewed and follow up asap. =)

File: README.md [hunk 1]
```diff
@@ -248,6 +248,21 @@ Hasura is an open source GraphQL engine that deploys instant, realtime GraphQL A
 - [Tutorial: Fullstack React Native with GraphQL and Authentication](https://blog.hasura.io/tutorial-fullstack-react-native-with-graphql-and-authentication-18183d13373a/)
 - [Guide: Hasura GraphQL Engine One-click App on DigitalOcean Marketplace](https://docs.hasura.io/1.0/graphql/manual/guides/deployment/digital-ocean-one-click.html)
 
+## Developer Productivity [↗](https://awesomestacks.dev/developer-productivity)
```
Comment (dzello) [inline README.md]: For the name, let's have one that mentions the tools and the use case, like "Monitoring Jenkins with Grafana"?

File: README.md [hunk 2]
```diff
@@ -248,6 +248,21 @@ Hasura is an open source GraphQL engine that deploys instant, realtime GraphQL A
...
+
+Boost your Productivty with GitHub, Jira, Jenkins, Grafana and Prometheus.
+
+- [GitHub](https://github.com/) - [🛠](https://stackshare.io/github) - The site to keep your code and open it to world.
+- [Jira](https://es.atlassian.com/software/jira) - [🛠️](https://stackshare.io/jira) - Agile software development.
+- [Jenkins](https://jenkins.io/) - [🛠️](https://stackshare.io/jenkins) - [🐙](https://github.com/jenkinsci/jenkins) - Open source automation server for CI/CD.
```
Comment (dzello) [inline README.md]: Let's have Jenkins, Grafana and Prometheus be the first tools since they are the key ones. GitHub and Jira could go afterwards then.

**Comment by dzello** [comment]
@OliverMorenoGonzalez Thanks for the submission! I left a few review comments - can you address those? Then I think we'll be ready to merge this in.

File: README.md (hunk 1 above)
Comment (OliverMorenoGonzalez) [reply README.md]: Sound good but It doesn't reflect the complete use case.
CI-CD Jenkins-GitHub-Jira monitored by Grafana?

Comment (dzello) [reply README.md]: Ah okay, I read the resources and I see what you mean. It looks like one use case is general Jenkins monitoring and another is the JIRA/GitHub workflow. The latter one seems the most primary so we could use that, how about his as the title: "Workflow automation with JIRA, GitHub and Jenkins"? It doesn't capture everything but the title space is limited. If you wanted, you could also break the Grafana/Prometheus stack out into another awesome stack, e.g. "Monitoring Jenkins with Grafana and Prometheus". The use case feels sufficiently different.

Comment (OliverMorenoGonzalez) [reply README.md]: Agree. In some hours I will address the request changes.

**Comment by OliverMorenoGonzalez** [comment]
Are you Ok with me doing this process?
Close this PR, commit again on a new branch, open another PR.

**Comment by dzello** [comment]
Yes, that works.

**Pull request closed without merging**